workspaces/acr/.changeset/version-bump-1-32-5.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-acr': major
```

Comment:
Sorry, I don't got before that the version update mean that bump for the plugin. This isn't required and the change is smaller then expected at all. I will close this PR.